testsuite/ext-tools/maintenance_crawler.rb

```diff
@@ -1,9 +1,8 @@
+#!/usr/bin/env ruby
```

Comment:
just change the shebang to `#! /usr/bin/ruby` and it will be possible to call it with simply `./crawler.rb`

Reply:
According to [Wikipedia](https://en.wikipedia.org/wiki/Shebang_(Unix)) the way Oscar did it is correct no need to introduce a white space there. It further states
> a shebang is the character sequence [...] at the beginning of a script.

FMPOV putting this at the beginning of the script (where it belong) is enough :wink:

Reply:
ok for the space

but i suggest to change `#!/usr/bin/env ruby` to `#! /usr/bin/ruby`

Reply:
With this we would loose portability. According to my research that's what the `env` command is somehow for in this context.

Reply:
I for example use different Ruby vesions:

```bash
❯ /usr/bin/ruby --version
ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [x86_64-linux-gnu]

❯ which ruby
/home/dgedon/.rvm/rubies/ruby-2.5.9/bin/ruby

❯ ruby --version
ruby 2.5.9p229 (2021-04-05 revision 67939) [x86_64-linux]
```

Reply:
you basically have no portability - inside a given version of ruby all gems tend to be incompatible... :trollface: 

also normally `usr/bin/ruby` should be a symlink to the default installed version.

but i checked, it works to run `./crawler.rb` with the `env` construct, so approving (although i don't see why it should be better)